code/__DEFINES/dcs/signals/signals_mob/signals_mob_main.dm

```diff
@@ -82,13 +82,10 @@
 	#define COMPONENT_UPPERCASE_SPEECH (1<<0)
 	// used to access COMSIG_MOB_SAY argslist
 	#define SPEECH_MESSAGE 1
-	// #define SPEECH_BUBBLE_TYPE 2
 	#define SPEECH_SPANS 3
-	// #define SPEECH_SANITIZE 4
 	#define SPEECH_LANGUAGE 5
-	/* #define SPEECH_IGNORE_SPAM 6
-	#define SPEECH_FORCED 7 */
-	#define SPEECH_RANGE 8
+	#define SPEECH_RANGE 9
+	#define SPEECH_SAYMODE 10
```

Comment:
also make sure to decrease these numbers to be sequential (1, 2, 3, 4, etc)

Reply:
The args for `COMSIG_MOB_SAY` are static.

`say(
  message, 
  bubble_type, 
  list/spans = list(), 
  sanitize = TRUE, 
  datum/language/language = null, 
  ignore_spam = FALSE, 
  forced = null, 
  filterproof = null, 
  message_range = 7, 
  datum/saymode/saymode = null)`
is the same as
`say(
  SPEECH_MESSAGE, 
  SPEECH_BUBBLE_TYPE, 
  SPEECH_SPANS, 
  SPEECH_SANITIZE, 
  SPEECH_LANGUAGE, 
  SPEECH_IGNORE_SPAM, 
  SPEECH_FORCED, 
  SPEECH_FILTERPROOF, 
  SPEECH_RANGE, 
  SPEECH_SAYMODE)`

This is one of the reasons why I didn't delete the commented out defines, since it shows that the defines are being used as an index for the argslist.

Reply:
re-add the defines then and uncomment them

Reply:
Implemented with https://github.com/tgstation/tgstation/pull/69844/commits/34f002cf689e7eda0ece30b791688b9a37e20fc3